tools/pipelines/templates/include-test-real-service.yml

```
@@ -340,12 +344,43 @@ stages:
               key: '"compat-version-installs" | "$(Agent.OS)" | "${{ parameters.testCommand }}" | "${{ variant.name }}"'
               path: ${{ parameters.testWorkspace }}/node_modules/@fluid-private/test-version-utils/node_modules/.legacy/
 
+        # Only check out tenants from the TRIPS tenant pool if we are running tests against ODSP
+        - ${{ if eq(parameters.stageId, 'e2e_odsp') }}:
+          # Retrieve a tenant from the TRIPS tenant pool
+          - task: AzureCLI@2
+            displayName: 'Log in to retrieve tenant credentials'
+            inputs:
+              azureSubscription: 'Fluid Framework Tests'
+              addSpnToEnvironment: true
```

When we were getting this working, I was wondering where some of the variables in the script below came from. Something like this might help that:
```suggestion
              # This injects environment variables into the below script. See https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/azure-cli-v2?view=azure-pipelines
              addSpnToEnvironment: true
```